swan-lake/learn/deployment/docker.md

```diff
@@ -466,7 +466,7 @@ This sample sends a greeting to the caller by getting the name from a text file.
 }
 ```
 
->**Note:** In addition to the above, if you want to copy driver files such as JDBC, you can create a Ballerina project, add following entires to its `Ballerina.toml` file, and change the path to the JDBC driver appropriately.
+>**Note:** In addition to the above, if you want to copy driver files such as JDBC, you can create a Ballerina project, add following entries to its `Ballerina.toml` file, and change the path to the JDBC driver appropriately.
```

Comment:
```suggestion
>**Note:** In addition to the above, if you want to copy driver files such as JDBC, you can create a Ballerina project, add the following entries to its `Ballerina.toml` file, and change the path to the JDBC driver appropriately.
```

Reply:
Done in .https://github.com/ballerina-platform/ballerina-dev-website/pull/1647